aes/src/ni.rs

```diff
@@ -71,36 +44,27 @@
 //! - [Intel AES-NI whitepaper](https://software.intel.com/sites/default/files/article/165683/aes-wp-2012-09-22-v01.pdf)
 //! - [Use of the AES Instruction Set](https://www.cosic.esat.kuleuven.be/ecrypt/AESday/slides/Use_of_the_AES_Instruction_Set.pdf)
 
-#![no_std]
-#![doc(
-    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
-    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg"
-)]
-#![warn(missing_docs, rust_2018_idioms)]
-
-pub use cipher;
-
-#[cfg(feature = "ctr")]
-pub use cipher::stream;
+#![allow(unsafe_code)]
```

Comment:
Personally I don't see much sense in using `deny(unsafe_code)` for a whole crate and then enabling it for selected modules.

Reply:
Removed the toplevel deny and moved it to the `soft` module in 18a2f0c